### PR TITLE
UI: rework webviews style/layout

### PR DESCRIPTION
**Pull request by justinmk3**
WIP. Low-effort improvements to webview styling, mainly to (1) match vscode style somewhat, (2) have some uniformity/consistency  to our webviews.

- Try to keep the "Submit" button above-the-fold.
- CSS was designed to use inheritance, let's use it.
- Add some uniformity to widths (80%).
- Attempt to match VSCode style.
- Use monospace for code inputs.

## TODO

- [ ] move common css to a `common.css` thing
- [ ] rethink placement of the "Submit" buttons so that they are always above-the-fold
- [ ] condense the forms some more (example: "Select a method" could be to the left of the dropdown, instead of above it, to save vertical space)

## Screenshot

webviewstyle

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: media/css/submitFeedback.css [hunk 1]
```diff
@@ -6,20 +6,37 @@ input[type='submit'] {
     cursor: pointer;
 }
 
+select {
+    background-color: var(--vscode-settings-selection-background);
+    color: var(--vscode-settings-textInputForeground);
+    width: 80%;
```
Comment (rli) [inline media/css/submitFeedback.css:12]: it's been a while, but any reason why 80% and not something like `100vw` (i.e. based on container width vs viewport width)?

File: media/css/submitFeedback.css [hunk 2]
```diff
@@ -6,20 +6,37 @@ input[type='submit'] {
...
 }
 
 input[type='submit']:disabled {
+    color: var(--vscode-settings-textInputForeground);
     cursor: default;
     opacity: 0.5;
 }
 
+input[type='text'] {
+    font-family: Menlo, Consolas, monospace;
```
Comment (rli) [inline media/css/submitFeedback.css:27]: probably: `var(--vscode-editor-font-family)`?

File: media/css/submitFeedback.css [hunk 3]
```diff
@@ -6,20 +6,37 @@ input[type='submit'] {
...
+    width: 80%;
+}
+
 input[type='submit']:hover {
     background-color: var(--vscode-button-hoverBackground);
+    color: var(--vscode-settings-textInputForeground);
 }
 
 input[type='submit']:disabled {
+    color: var(--vscode-settings-textInputForeground);
```
Comment (rli) [inline media/css/submitFeedback.css:21]: promote text color color to outer block:
```css
input[type='submit'] {
    color: var(--vscode-button-foreground);
}

input[type='submit']:hover {
    background-color: var(--vscode-button-hoverBackground);
}

input[type='submit']:disabled {
```

**Comment by bryceitoc9** [comment]
Just as a sanity check, can you post screenshots from a couple other themes as well?

**Comment by hunterwerlla** [comment]
Exciting to see :D

**Comment by justinmk3** [comment]
closing in favor of https://github.com/aws/aws-toolkit-vscode/pull/2052

**Pull request closed without merging**